tests/unit/backend/chalice/api_server/test_v1_auth.py

```diff
@@ -45,15 +45,17 @@ def test__auth_flow(self):
             self.assertEqual(args["client_id"], self.auth_config.client_id)
             self.assertEqual(args["response_type"], "code")
             self.assertTrue("/dp/v1/oauth2/callback" in args["redirect_uri"])
-            self.check_set_cookie_is_secure(response.headers["Set-Cookie"])
+            self.check_set_cookie_is_secure(response.headers["Set-Cookie"], ["HttpOnly", "Secure", "SameSite=Lax"])
```

Comment:
Can we add comment that this is for asserting `session` cookie and the next block is for `cxguser` cookie please? At a glance, it's hard to tell which cookie we're asserting 😄 Thank you!

Reply:
Good idea. It is confusing. Done!